jtec-core/pom.xml

```diff
@@ -39,6 +39,16 @@
             <artifactId>gson</artifactId>
             <version>2.9.0</version>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.javaparser</groupId>
+            <artifactId>javaparser-core</artifactId>
+            <version>3.24.8</version>
+        </dependency>
```

Comment:
Make sure to also include this in the shaded packages of the fat JAR of the `jtec-agent` (see: https://github.com/tum-i4/JTeC/blob/main/jtec-agent/pom.xml#L77)